### PR TITLE
Base of website protocol

### PR DESCRIPTION
**Pull request by TPGamesNL**
Carried over the protocol between judge and runner. Modifications are renaming references, making website the server and judge the client, making it a one-to-one connection

### CODE REVIEW

File: judgequeuer.py [hunk 1]
```diff
@@ -24,47 +21,16 @@
...
-        handle_connection(Connection(addr[0], addr[1], client_sock, threading.Lock()))
+JUDGE_PROTOCOL_HOST = "localhost"
+JUDGE_PROTOCOL_PORT = 12345
+WEBSITE_PROTOCOL_HOST = "localhost"
+WEBSITE_PROTOCOL_PORT = 30000
 
 
 async def main():
-    threading.Thread(target=establish_connection, daemon=True).start()
+    judge_protocol_handler.start_handler(JUDGE_PROTOCOL_HOST, JUDGE_PROTOCOL_PORT)
```
Comment (andreitudorp) [inline judgequeuer.py:31]: Should we make classes for the handlers?

**Review by andreitudorp — COMMENTED** [review]
One small comment

**Review by andreitudorp — APPROVED** [review]
Literalmente, perfect! 👍

**Comment by TPGamesNL** [comment]
TODO after merge, in other PR: remove code duplication

**Pull request merged**